readthedocsext/theme/templates/projects/partials/header.html

```diff
@@ -20,6 +20,14 @@
 {% endcomment %}
 
 {% block project_header %}
+  {# Render all the notifications attached to the project.#}
+  {% for notification in project.notifications.all %}
+    <div class="ui message">
+      <i class="fa-solid {{ notification.get_message.get_display_icon_classes }}"></i>
```

Comment:
And same changes here

```suggestion
      <i class="{{ notification.get_message.get_display_icon_classes }} icon"></i>
```